src/actions/breakpoints/syncBreakpoint.js

```diff
@@ -40,12 +59,14 @@ export async function syncClientBreakpoint(
   // early return if breakpoint is disabled, send overrides to update
   // the id as expected
   if (pendingBreakpoint.disabled) {
-    const newLocation = await sourceMaps.getOriginalLocation(generatedLocation);
+    const newLocation = await sourceMaps.getOriginalLocation(
+      scopeGeneratedLocation
```

Comment:
another nit:

maybe we want some smaller functions here, like each case is it's own function. the top vars are defined where they're used